packages/patternfly-3/patternfly-react/src/components/DateTimePicker/TimePicker.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PickTimeTable from './Components/PickTimeTable';
+import PickTimeClock from './Components/PickTimeClock';
+
+class TimePicker extends React.Component {
+  state = {
+    time: new Date(this.props.time),
+    typeOfTimeTable: '',
+    isTimeTableOpen: false
+  };
+
+  toggleTimeTable = (type = null) => {
```

Comment:
Since we default the type to `null`, this could probably be simplified to:

```js
toggleTimeTable = (type = null) => {
  this.setState({
    type,
    isTimeTableOpen: !this.state.isTimeTableOpen
  });
}
```